### PR TITLE
fix potential bug risks and anti-patterns

### PR DESCRIPTION
**Pull request by tusharnankani**

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@tusharnankani thanks for contributing, will have a look at it when I have time 😄

**Comment by gautamkrishnar** [comment]
@tusharnankani are you contributing as the part of DWOC (https://devscript.tech/woc/) ?

**Comment by tusharnankani** [comment]
> @tusharnankani are you contributing as the part of DWOC (https://devscript.tech/woc/) ?

Yes.

File: socli/search.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def socli_interactive_windows(query):
                                     socli.printer.print_warning(" No more answers found for this question. Exiting...")
                                     sys.exit(0)
                                 continue
-                            elif qna in ["b", "B"]:
+                            if qna in ["b", "B"]:
```
Comment (gautamkrishnar) [inline socli/search.py:246]: @tusharnankani can you please explain why you changed elif to if?

File: socli/search.py [hunk 2]
```diff
@@ -253,7 +253,7 @@ def socli_interactive_windows(query):
                                 print("-------\n" + answer + "\n-------\n")
                                 cnt = cnt - 1
                                 continue
-                            elif qna in ["o", "O"]:
+                            if qna in ["o", "O"]:
```
Comment (gautamkrishnar) [inline socli/search.py:256]: Here as well?

Comment (gautamkrishnar) [reply socli/search.py:256]: And all other places?

**Review by gautamkrishnar — CHANGES_REQUESTED** [review]
@tusharnankani please answer the comments

**Comment by gautamkrishnar** [comment]
@tusharnankani I am not sure why you modified lots of if statements, there is a difference between using if-else and if only.

**Comment by tusharnankani** [comment]
@gautamkrishnar 

Apologies for the confusion, I should have written what changes I made, and why. This PR includes:
- The else / elif block here is unnecessary because the leading if block has a `continue` statement, or any other statement, like `break`, or `return`. These come under anti-patterns,
  - These statements causes the control flow to be disrupted, making the else / elif block here unnecessary. This doesn't mean you can not use it, but it is recommended to refactor this for a better readability.
- Used `sys.exit()`, 
- Import alias same as the original package name
- Simplified if expression - `bool(namespace.json)`
- remove unnecessary `dict()` calls
  - It is slower to call e.g. `dict()` than using the empty literal - `{}`, because the name dict must be looked up in the global scope in case it has been rebound. Same for the other two types like list() and tuple().

**Review by gautamkrishnar — APPROVED** [review]

**Comment by gautamkrishnar** [comment]
@tusharnankani thanks a lot for the contribution 🎉 

@all-contributors please add @tusharnankani for code

**Pull request merged**